docs/user/migration/external-name-alias.md

```diff
@@ -0,0 +1,148 @@
+# Migration Guide for External Name Alias's Behavior Change
+
+## Context
+
+Istio with 1.21 version has changed the behavior of the Service of type ExternalName.
+Previously, for the Service of type ExternalName a separate `cluster` in the Envoy config was created.
+With the change, the Service of type ExternalName is treated as an alias of the Service that it points to.
+This change has been introduced to align the behavior of Istio with handling of the Service of type ExternalName with the Kubernetes behavior.
+Since it caused some issues for our users, we introduced a new annotation "disable-external-name-alias" to disable this behavior.
```

Comment:
```suggestion
Since it caused some issues for our users, we introduced a new annotation, **disable-external-name-alias**, to disable this behavior.
```